semgrep_output_v1.atd

```diff
@@ -713,6 +712,17 @@ type cli_timing <ocaml attr="deriving show"> = {
     ?max_memory_bytes : int option;
   }
 
+type contribution <ocaml attr="deriving show"> = {
```

Comment:
maybe some link to some documentation about what a "contribution" means, or an example.

Reply:
Added a link to the public usage-limit docs. I suspect we'll update those docs at some point soon with more details about how the contributor/contribution count works.

Reply:
Oh wow, I completely misinterpreted what it was for at the beginning. I thought it was just for some internal stats about who was using semgrep (well it is, but the objective is now clearer).

Reply:
How usage limit was enforced before if we were not sending this info? Or we were sending it but it's just it was not specified in the ATD file?

Reply:
Before this, we would send the commit info (commit hash, author name, author email) as scan metadata when the CLI first created the scan. However, this would only account for a single commit, and it could be overwritten by environment variables (those had a higher priority).

The new approach is to also use `git log` as a source of truth. I say "also" because the CLI will still have the option to configure the latest commit hash/author/email with environment variables.